### DISCUSSION TITLE
IOS Server Not reachable

### DISCUSSION THREAD

**Discussion by ghanjiboy** [immich-app/immich#11214] [Q&A]
### The bug

On one of 3 iphones in my house, whenever I try to put the server address in, it always says "Server not reachable"   My current setup has immich running in a docker container with nginx in front of it and a lets encrypt certificate.

I am able to access the same server from 2 other Iphones and 1 android.

I've tried to turn on troubleshooting and increased the logs from this iPhone, when I do that, I see in the log "no host specified"

I am able to access the same server URL from chrome or safari from this device - so I am not sure whats going on.  IOS is updated on this device and I am running the latest immich.

Is there anything else I can do to troubleshoot? 

### The OS that Immich Server is running on

Debian 11 with docker 20.10

### Version of Immich Server

v1.109.2

### Version of Immich Mobile App

v1.109.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
docker run -d --restart always \
  --name=immich \
  -e PUID=1000 \
  -e PGID=1000 \
  -e TZ=America/New_York \
  -e DB_HOSTNAME=192.168.1.14 \
  -e DB_USERNAME=<USER> \
  -e DB_PASSWORD=<PASSWORD> \
  -e DB_DATABASE_NAME=immich \
  -e REDIS_HOSTNAME=192.168.1.14 \
  -e REDIS_PORT=6379 `#optional` \
  -e MACHINE_LEARNING_GPU_ACCELERATION= `#optional` \
  -e MACHINE_LEARNING_WORKERS=1 `#optional` \
  -e MACHINE_LEARNING_WORKER_TIMEOUT=120 `#optional` \
  -p 8080:8080 \
  -v /pictures/config:/config \
  -v /pictures/photos:/photos \
  --restart unless-stopped \
  ghcr.io/imagegenius/immich:latest
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1.  Install immich on IOS
2.  Run Immich
3.  Enter https://URL
4.  Recieved error message "Server not reachable"
```

**Comment by bo0tzz** [comment]
What is the address you're entering into the app? Can you post the full app logs?

**Reply by ghanjiboy** [reply]
So I am using something like https://immich.myurl.com  which hits an nginx server and proxies the request to http://192.168.1.14:8080

This morning, I put http://192.168.1.14:8080 in and it went to the next screen to login.  Then I went back and put in the original https address and then it started working again.

I cant explain this sequence of events - but now it seems to be working - even after days of just using the https URL

**Comment by hitzhangjie** [comment]
same problem, i can access via mobile Safari, and i have toggled allow local network. and i could see the immich app login last seen in admin setting

**Reply by alextran1502** [reply]
Are you login using local ip? Is there any vpn in between?

**Reply by tho-nk** [reply]
Hi, same here.
I restart my iPhone then put: http://local-ip:2283/
It manages to find the server but I still got : 
"Your app major version is not compatible with the server!"
But it works!

Everything works well on safari/chrome on Iphone

**Comment by vvsrri** [comment]
I am also facing same error, "Server is not reachable". I am using Fedora 41 for immich server.
But surprisingly, when I tried for first time it worked smoothly. It is not working second time onwards. I tried removing the app and reinstalled and restarted docker. No luck.
I am using iphone 11 and ios 18.3.2 and immich v1.131.3.

**Comment by Qlub53** [comment]
@alextran1502  It's a bug in the iOS app. The app should've prompted the user for permission to access "Local Network" but it never did.
We as iOS users have no way to grant such an access unless prompted.

**Reply by Qlub53** [reply]
> @alextran1502 It's a bug in the iOS app. The app should've prompted the user for permission to access "Local Network" but it never did. We as iOS users have no way to grant such an access unless prompted.

Didn't notice #19302 and #19285
Please disregard my previous comment.

**Reply by qbirdee** [reply]
Having the same problem, the local network permission is not being surfaced on iPad OS.  Any way to force the permission request from Immich so it shows up in settings?  Thanks.

**Reply by bo0tzz** [reply]
Update to the latest version of the app.

**Reply by qbirdee** [reply]
Yes!  Updated today and now finally prompted for local network and working. Thanks!
